VideoLocker/src/main/java/org/edx/mobile/util/IOUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.edx.mobile.util;
+
+import android.support.annotation.NonNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+
+import okio.Okio;
+
+public class IOUtils {
+    @NonNull
+    public static String toString(@NonNull InputStream in, @NonNull Charset charset) throws IOException {
+        return Okio.buffer(Okio.source(in)).readString(charset);
```

Comment:
> Any files or API responses we read would presumably be be UTF-8 encoded.

Probably, but not necessarily true. Really, we should be reading the content type of the response headers.

I think it's a problem that the callers were using `Charset.defaultCharset()` instead of an explicit charset, something that would be nice to fix. In any case, the caller should be determining the charset 100% of the time, because only the caller knows what charset is appropriate. I don't think the utility method should have a default.